src/admin/custom-give.ts

```diff
@@ -19,16 +20,14 @@ registerCommand(
         Material.getMaterial(args[0]);
       const id = Number.parseInt(args[1]);
 
-      const name = args[2] || undefined;
-      const modelId = Number.parseInt(args[3]) || id;
+      const name = args[2];
 
       if (!type) return;
 
       const item = new CustomItem({
         id: id,
         type: type,
-        modelId: modelId,
-        name: name,
+        name: name ? text(name) : undefined,
```

Comment:
We could maybe add check if the name starts with "vk." and use translate() component? This would make this command more usefull